src/ring_theory/derivation.lean

```diff
@@ -131,34 +131,14 @@ variables (f : M →ₗ[A] N)
 
 /-- We can push forward derivations using linear maps, i.e., the composition of a derivation with a
 linear map is a derivation. Furthermore, this operation is linear on the spaces of derivations. -/
-def comp : derivation R A M →ₗ[R] derivation R A N :=
+def _root_.linear_map.comp_der : derivation R A M →ₗ[R] derivation R A N :=
```

Comment:
Wow! Since when does that work?! Nice.

Reply:
Or maybe not... the ability to put one-off definitions in other namespaces I guess increases the temptation to not organise things well... We'll see. :-)

Reply:
Given that this function is intended to be used with dot notation, it might be more readable to explicitly include the linear map argument, rather than pulling it in via `variables`.

Reply:
I think elsewhere in the library we have these as `comp_left` and `comp_right` (i.e. a fixed argument for one side of the composition, and then a bundled map with respect to the other). I guess here the `_der` is enough. But what if I wanted to know this was linear in `f`, as well?

Reply:
@ocfnash decided to PR this modification independently before this PR was merged, with his PR: #7715. It appears a modification here because I did not merge the master branch yet. All these comments belong to that PR, maybe you want to ask him to reopen it?